src/components/side_panel/chart/building_blocks/data_series/data_series.xml

```diff
@@ -1,6 +1,25 @@
 <templates>
   <t t-name="o-spreadsheet.ChartDataSeries">
-    <Section class="'o-data-series'" title="title">
+    <Section class="'o-data-series'">
+      <div class="d-flex flex-row o-section-title">
+        <span class="w-100">
+          <t t-esc="title"/>
+        </span>
+        <span
```

Comment:
According to the props, `onFlipAxis` could be undefined, if so these icons should be hidden IMO